models/models_test.go

```diff
@@ -94,12 +94,36 @@ func (s *ModelsSuite) createCampaign(ch *check.C) Campaign {
 	return c
 }
 
-func (s *ModelsSuite) TestGetUser(c *check.C) {
+func (s *ModelsSuite) TestGetUserExists(c *check.C) {
 	u, err := GetUser(1)
 	c.Assert(err, check.Equals, nil)
 	c.Assert(u.Username, check.Equals, "admin")
 }
 
+func (s *ModelsSuite) TestGetUserDoesNotExist(c *check.C) {
+	u, err := GetUser(100)
+	c.Assert(err, check.Equals, gorm.ErrRecordNotFound)
+	c.Assert(u.Username, check.Equals, "")
+}
+
+func (s *ModelsSuite) TestGetUserByAPIKeyWithExitingAPIKey(c *check.C) {
```

Comment:
Hi there!

Did you mean TestGetUserByAPIKeyWithExi**s**tingAPIKey? 😄

Reply:
(note the "s" in Existing)

Reply:
Oops, sorry and thanks for pointing it out! 🙇